test/libcommonserver/io/testisfileaccessible.cpp

```diff
@@ -30,10 +30,17 @@ namespace KDC {
 
 void TestIo::testIsFileAccessible() {
     const TemporaryDirectory temporaryDirectory;
-    const SyncPath sourcePath = _localTestDirPath / "big_file_dir/test_big_file.mov";
+    const SyncPath sourcePath = temporaryDirectory.path / "test_big_file.txt";
+    const SyncPath destPath = temporaryDirectory.path / "test_big_file_copy.txt";
 
-    std::shared_ptr<LocalCopyJob> copyJob =
-        std::make_shared<LocalCopyJob>(sourcePath, temporaryDirectory.path / "test_big_file.mov");
+    // Create a 100 MB file
+    std::ofstream sourceFile(sourcePath);
```

Comment:
```suggestion
    {
       std::ofstream sourceFile(sourcePath);
       sourceFile << std::string('b', 10000000);
     }
```
- The destructor of `std::ofstream` automatically closes the file.
- The allocation and destruction of the big `std::string` should be faster than the loop.

Reply:
Done